### ISSUE TITLE
Minimum on the education section of job poster

### ISSUE THREAD

**Issue by gobyrne** [GCTC-NTGC/design-gc-digital-talent#28]
In the minimum experience or education step, we imply that there's no difference whether candidates meet the education criteria or the applied work experience option. However, this isn't exactly true. In some cases managers may request talent that meets the strict education requirement (and not the work experience).

On the business side, we don't think it's practical to remove this option as it could be out of the managers hands. I'll bring it up with Recruitment team though, maybe they know something I don't. 

We might want to revise the text on this page to give preference to the education rather than education.

![Image](https://github.com/GCTC-NTGC/design-gc-digital-talent/assets/2494551/4afd2282-910f-478c-9a28-2050238a1112)

**Comment by substrae** [comment]
@gobyrne This is Lauren's call, not ours. We've specifically designed this interaction with the intent of moving the needle in terms of equivalent experience being accepted, so I'm reticent to change anything on this page until after we've sat down together to discuss.

**Comment by gobyrne** [comment]
Agreed, I'll bring to Lauren, then recruitment team after, I think.

**Comment by RM-1978** [comment]
As promised, here is the screen capture and edited text that Lauren has seen from the editors as part of the copy project: 

[Application education requirement_edited and proofed.docx](https://github.com/GCTC-NTGC/design-gc-digital-talent/files/13865629/Application.education.requirement_edited.and.proofed.docx)

**Comment by gobyrne** [comment]
@NienkeBr did we already address this?

**Comment by NienkeBr** [comment]
My first day we looked at this and changed it to "If both apply to you, select the education criteria."

![Image](https://github.com/user-attachments/assets/5aef10b2-cc19-42db-84e3-cb263d84ebc1)
